static/temp/recipe_preferences.json

```diff
@@ -0,0 +1,32 @@
+{
+  "setYourPreferences": {
+    "all": [
+      { "name": "Dietary" },
+      { "name": "Allergy" },
+      { "name": "Meal type" },
+      { "name": "Cooking time" }
+    ],
+    "preference": [
+      {
+        "filterName": "Dietary",
+        "items": ["Gluten-free", "Dairy-free", "Vegan"]
```

Comment:
rename `items` to `values`. This change should be for everywhere where `items` exists.

Reply:
done